src/Compatibility/Microsoft.DotNet.ApiCompatibility/ValidationSuppressions/Suppression.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+using System;
+using System.Globalization;
+
+namespace Microsoft.DotNet.ValidationSuppression
+{
+    /// <summary>
+    /// Represents a Suppression for a validation error.
+    /// </summary>
+    public class Suppression : IEquatable<Suppression>
```

Comment:
What are your thoughts of adding a field to mark if it is a baseline suppression (running against previous package version) instead of encoding it as part of the `Left` string? I think that way when we include this as part of API Compat, we wouldn't need to use the same encoding for the error messages when using `Left` ID.

Reply:
That is a good point, I can try doing that and see how it looks.